Python/Product/Profiling/PythonProfilingPackage.cs

```diff
@@ -286,13 +286,16 @@ protected override int CreateToolWindow(ref Guid toolWindowType, int id)
 
         private static void ProfileStandaloneTarget(SessionNode session, StandaloneTarget runTarget, bool openReport) {
             LaunchConfiguration config;
-            if (runTarget.PythonInterpreter != null) {
+            if (runTarget.PythonInterpreter != null && runTarget.PythonInterpreter.Id != "Global|Other...") {
```

Comment:
I would actually nest these conditionals because they are checking two different things, and the final `else` is really only supposed to run if the interpreter is null. So it's technically correct, but it's easy to introduce a bug here.

Something like this:

```csharp
if (runTarget.PythonInterpreter != null) {
  if (runTarget.PythonInterpreter.Id != "Global|Other...") {
    ...
  } else {
    ...
  }
else {
  config = new LaunchConfiguration(null);
}
```

Reply:
(and again, `"Global|Other..."` should not be hardcoded here, it should be checking against a static string)